docs/extend/plugins_volume.md

```diff
@@ -12,10 +12,10 @@ parent = "mn_extend"
 
 Docker volume plugins enable Docker deployments to be integrated with external
 storage systems, such as Amazon EBS, and enable data volumes to persist beyond
-the lifetime of a single Docker host. See the [plugin documentation](plugins.md)
+the lifetime of a single Docker host. See the [plugin documentation](plugin_api.md)
```

Comment:
hm, alright. point was that the page had another "How to write a plugin" heading, so it felt as linking in a "loop". I'll change back